encrypted-media-respec.html

```diff
@@ -1498,17 +1498,19 @@
             The time represented by the <a def-id="expiration"></a> attribute MUST be earlier than the current time.
             All other keys in the session MUST have this status.
           </dd>
+          <dt>output-restricted</dt>
+          <dd>
+            There are output restrictions associated with the key which cannot currently be met.
+            The application should avoid using streams which will trigger the output restrictions associated
+            with the key.
+          </dd>
           <dt>output-downscaled</dt>
           <dd>
             The key is not currently usable to decrypt <a def-id="media-data"></a> at full quality (e.g. resolution) because its output requirements cannot currently be met.
-            Media data decrypted with this key may be presented at a lower quality (e.g. resolution).<br/>
+            Media data decrypted with this key will be presented at a lower quality (e.g. resolution).<br/>
```

Comment:
nit: "will" seems more absolute than we mean. It will only be lower if necessary to meet the output requirements. Perhaps, "will... if necessary to meet the output requirements."